src/gui/EditWidgetIcons.cpp

```diff
@@ -223,8 +223,15 @@ void EditWidgetIcons::fetchFavicon(const QUrl& url)
     }
 
     m_httpClient->setConnectingTimeOut(5000, [this]() {
-        resetFaviconDownload();
-        MessageBox::warning(this, tr("Error"), tr("Unable to fetch favicon."));
+        if (m_url.startsWith("http://")) {
+            resetFaviconDownload();
+            MessageBox::warning(this, tr("Error"), tr("Unable to fetch favicon."));
+        } else {
+            m_url = m_url.replace("https","http");
```

Comment:
@lopopolo nice catch! I've pushed a fix for that

@hifi what if user's url doesn't hava a protocol? Like `cmd://firefox www.google.com` ?